### ISSUE TITLE
composer network commands fail ungracefully

### ISSUE THREAD

**Issue by davidkel** [hyperledger/composer#2623]
```
ubuntu@ubuntu-xenial:~$ composer network ping
/home/ubuntu/.nvm/versions/node/v8.9.1/lib/node_modules/composer-cli/node_modules/yargs/yargs.js:1100
      else throw err
           ^

TypeError: Path must be a string. Received undefined
    at assertPath (path.js:28:11)
    at Object.join (path.js:1251:7)
    at FileSystemCardStore._cardPath (/home/ubuntu/.nvm/versions/node/v8.9.1/lib/node_modules/composer-cli/node_modules/composer-common/lib/cardstore/filesystemcardstore.js:63:21)
    at FileSystemCardStore.get (/home/ubuntu/.nvm/versions/node/v8.9.1/lib/node_modules/composer-cli/node_modules/composer-common/lib/cardstore/filesystemcardstore.js:71:42)
    at BusinessNetworkConnection.connect (/home/ubuntu/.nvm/versions/node/v8.9.1/lib/node_modules/composer-cli/node_modules/composer-client/lib/businessnetworkconnection.js:447:31)
    at Function.handler (/home/ubuntu/.nvm/versions/node/v8.9.1/lib/node_modules/composer-cli/lib/cmds/network/lib/ping.js:40:42)
    at Object.module.exports.handler (/home/ubuntu/.nvm/versions/node/v8.9.1/lib/node_modules/composer-cli/lib/cmds/network/pingCommand.js:26:35)
    at Object.runCommand (/home/ubuntu/.nvm/versions/node/v8.9.1/lib/node_modules/composer-cli/node_modules/yargs/lib/command.js:228:22)
    at Object.parseArgs [as _parseArgs] (/home/ubuntu/.nvm/versions/node/v8.9.1/lib/node_modules/composer-cli/node_modules/yargs/yargs.js:1013:30)
    at Object.runCommand (/home/ubuntu/.nvm/versions/node/v8.9.1/lib/node_modules/composer-cli/node_modules/yargs/lib/command.js:194:45)
ubuntu@ubuntu-xenial:~$
```

**Comment by davidkel** [comment]
Also get same results in composer network list

**Comment by avoicule** [comment]
got the same error

**Comment by davidkel** [comment]
- [ ] composer network list
- [ ] composer network reset
- [ ] composer network loglevel

**Comment by mbwhite** [comment]
Resolved in #2736

**Comment by andrew-coleman** [comment]
resolved by https://github.com/hyperledger/composer/pull/2731
